openreview/conference/templates/programchairWebfield.js

```diff
@@ -518,12 +518,14 @@ var getUserProfiles = function(userIds, reviewerBidCounts, areaChairBidCounts, a
         profileMap[profile.id] = {
           id: profile.id,
           name: name,
+          gender: profile.content.gender,
```

Comment:
I'm not sure about exporting this data in all the PC consoles. I know this data is public but we don't show in the public profile so the owner of the profile doesn't know.